pkg/virtctl/convert-machine-type/convert-machine-type-job/update_machine_type.go

```diff
@@ -0,0 +1,152 @@
+package convertmachinetypejob
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	k8sv1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	k6tv1 "kubevirt.io/api/core/v1"
+)
+
+// using these as consts allows us to easily modify the program to update as newer versions are released
+// we generally want to be updating the machine types to the most recent version
+const (
+	LatestMachineTypeVersion           = "rhel9.2.0"
```

Comment:
As a start, let's get this via a parameter

Reply:
Yes.